### PR TITLE
Fixes 1217562 - Update dependencies to more final versions

### PR DESCRIPTION
**Pull request by st3fan**
This patch updates the following dependencies to more final versions:

* Alamofire
* XCGLogger
* Snap

The first two used a beta version that is causing trouble with app store submission. (iTunes Connect will reject versions like `3.0b2`)

The latter was pointing to a swift-2.0 branch that is not used anymore, so I updated it to their last stable version.

### CODE REVIEW

**Comment by rnewman** [comment]
You should also run `./update.sh` and test+commit the changes to `Cartfile.resolved`.

**Comment by Johennes** [comment]
It seems like the `swift-2.0` branch of Alamofire was merged and deleted upstream which means that the `Cartfile` currently points to an invalid reference.

File: Cartfile [hunk 1]
```diff
@@ -6,10 +6,10 @@ github "kif-framework/KIF"          "master"
 github "ZaBlanc/RaptureXML"         "master"
 github "mozilla/readability"        "master"
 github "rs/SDWebImage"              "master"           # To get the fix for libwebp
-github "Masonry/Snap"               "Swift-2.0"
+github "Masonry/Snap"               "0.16.0"
```
Comment (ikesyo) [inline Cartfile]: [0.17.0](https://github.com/SnapKit/SnapKit/releases/tag/0.17.0) is released 2 days ago, and the repo is renamed / transferred to [SnapKit/SnapKit](https://github.com/SnapKit/SnapKit).

**Comment by rnewman** [comment]
@st3fan: ping on this? Seems like we could do this one by one if Alamofire is giving us trouble…

**Pull request closed without merging**